template/README.md

```diff
@@ -1,137 +1,163 @@
-# Starter template for `fredrikhgrelland/hashistack`
+<!-- markdownlint-disable MD041 -->
+<p align="center">
+ <img width="100px" src="https://www.svgrepo.com/show/58111/cube.svg" align="center" alt="Vagrant-hashistack" />
+ <h2 align="center">Vagrant-hashistack Template</h2>
+ <p align="center">Starter template for <a href="https://github.com/fredrikhgrelland/vagrant-hashistack">fredrikhgrelland/vagrant-hashistack</a></p>
+<p align="center">
+    <a href="https://github.com/fredrikhgrelland/vagrant-hashistack-template/actions">
+      <img alt="Build" src="https://github.com/fredrikhgrelland/vagrant-hashistack-template/workflows/CI/CD/badge.svg" />
+    </a>
+    <a href="https://github.com/fredrikhgrelland/vagrant-hashistack/releases">
+      <img alt="Releases" src="https://img.shields.io/github/v/release/fredrikhgrelland/vagrant-hashistack-template?label=latest%20version" />
+    </a>
+    <a href="https://github.com/fredrikhgrelland/vagrant-hashistack/commits">
+      <img alt="Updated" src="https://img.shields.io/github/last-commit/fredrikhgrelland/vagrant-hashistack-template?label=last%20updated" />
+    </a>
+    <br />
+    <br />
+    <p align="center">
+      <a href="https://github.com/fredrikhgrelland/vagrant-hashistack-template/generate" alt="Clone Template">
+            <img src="https://img.shields.io/badge/Github-Clone%20template-blue?style=for-the-badge&logo=github" />
+        </a>
+    </p>
+</p>
+
```

Comment:
Just an offer (if will be applied, do NOT forget to remove `/` escape char)
```suggestion
## Start

To start to develop terraform module:

/```text
make dev
/```

Follow [Iteration of the development process](#iteration-of-the-development-process) for more information.
```

Reply:
Hm, I agree with this, but there will be a full "how-to" later, so I think it's better to put it in that PR

Reply:
Could you create an issue and resolve this conversation? @pdmthorsrud

Reply:
Yup, will do in like 15 minutes, my day is a little packed!

Reply:
Added #272, resolving